src/offline/mod_offline.erl

```diff
@@ -463,49 +470,60 @@ pop_messages(HostType, JID) ->
             Other
     end.
 
--spec remove_user(mongoose_acc:t(), jid:luser(), jid:lserver()) -> mongoose_acc:t().
-remove_user(Acc, LUser, LServer) ->
+-spec remove_user(Acc, Params, Extra) -> {ok, Acc} when
+    Acc :: mongoose_acc:t(),
+    Params :: map(),
+    Extra :: map().
+remove_user(Acc, #{jid := #jid{luser = LUser, lserver = LServer}}, #{host_type := HostType}) ->
     HostType = mongoose_acc:host_type(Acc),
```

Comment:
I think we can remove this line since `HostType` is already bound in function's head.